README.md

```diff
@@ -45,5 +45,5 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/[USERN
 
 ## License
 
-The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
+The gem is available as an open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
```

Comment:
@kboygit, 'open source' in this context is acting as an adjective, not a noun - this should not be changed.